cljfmt/src/cljfmt/diff.clj

```diff
@@ -9,13 +10,18 @@
 (defn unlines [ss]
   (str/join "\n" ss))
 
+(defn to-absolute-path [filename]
```

Comment:
Not sure if this defn should go here. Else I'd suggest a tiny `utils` namespace.

Reply:
Do we need it in another namespace? If not, just make it private. All other functions we don't want as part of the public API need to be private as well, like `lines` and `unlines`.